### ISSUE TITLE
Feature request: multi-user accessible lists / groups

### ISSUE THREAD

**Issue by Simonjohanning** [SecUSo/privacy-friendly-shopping-list#59]
First of all, thank you very much for your work!
The app looks promising and quite what I am looking for.
However, I live in a shared flat, and in order to coordinate our shopping we require a list that can be accessed by several users / our group (as in out-of-milk).
Unfortunately, such a feature is a must for our shopping list (too many cases of three breads or 6 liters of milk), and I would be delighted if a future version would include this feature.
Thanks a lot in advance!

**Comment by supertobi** [comment]
Yes, that's the main reason we use a digital shopping list in our family. At the moment we use "Die Einkaufsliste"[1] but I would prefer an open source solution.

[1] https://play.google.com/store/apps/details?id=de.markusliebschner.thatlist

**Comment by petermayer** [comment]
Just as a quick update, we are looking into making syncing and sharing possible for our privacy friendly apps, but we are not far enough to make any promisses.

**Comment by sebastianschauenburg** [comment]
Sounds like a plan (I'd like this feature as well), thanks for sharing!
If you want a "quick" way (without too much work), importing/exporting would initially suffice. Then user A could send a shopping list to user B (via e-mail for example) and they could work through that. If syncing is wanted, some 3rd platform (like Nextcloud) could help.

**Comment by Rainer-Lang** [comment]
Any news on this?

**Comment by petermayer** [comment]
Yes and no.

**On the YES side:** We have made a (for us) big first step in that direction by releasing our [Backup App](https://github.com/SecUSo/privacy-friendly-backup). This allows exporting and importing data from basically all PFAs that have any form of user data, including also this one, our shopping list app.

**On the NO side:** We have not implemented sharing of individual shopping lists with other users yet. Having the backup functionality integrated in most apps, sharing of individual items in apps (for this app individual shopping lists) is a potential next step, but again I cannot make any promises. Once we tackle this, we want to do it in a way that would allow it to work in an API fashion, like our backup integration does so that it scales for all apps and we do not have to implement sharing features for each app (where it makes sense). This means it is no quick thing to do.

I understand this might not be the answer you were hoping for, but I hope you can understand our situation and the limited resources we have.
